### ISSUE TITLE
Test description says “date must be in the past” (i.e. `< NOW`), but test itself checks `≤ NOW`

### ISSUE THREAD

**Issue by andylolz** [IATI/IATI-Rulesets#40]
6d20f562 makes `date_order` tests inconsistent with their descriptions.

**Comment by andylolz** [comment]
NB my proposed solution would *not* be:

```patch
diff --git a/iatirulesets/__init__.py b/iatirulesets/__init__.py
index ed70a05..9e899ea 100644
--- a/iatirulesets/__init__.py
+++ b/iatirulesets/__init__.py
@@ -57,7 +57,7 @@ class Rules(object):
         if less is None or more is None:
             return None
         else:
-            return less <= more
+            return less < more
 
     def _regex_matches(self, case):
         return [ re.search(case['regex'], get_text(path_match)) for path_match in self.path_matches ]
```

…because while that might be more consistent, it is definitely wrong.

**Comment by andylolz** [comment]
Fixed in 77d5f95d.
